specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/CommonDefinitions.json

```diff
@@ -1614,6 +1614,208 @@
         }
       }
     },
+    "ManagedHttpRoutePatch": {
+      "description": "A managed http route to update",
+      "type": "object",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Application-specific metadata in the form of key-value pairs."
+        }
+      }
+    },
+    "ManagedHttpRoute": {
```

Comment:
Shouldn't this be ManagedHttpRouteConfig?

Reply:
ManagedEnvironment and ManagedCerfiticates are the full names of APIs and classes. We don't call this ManagedHttpRoute anywhere else, it's called HttpRouteConfig.

Reply:
Changed